pkg/projectfile/projectfile.go

```diff
@@ -1005,7 +1005,7 @@ func createCustom(params *CreateParams, lang language.Language) (*Project, error
 	}
 
 	content := params.Content
-	if content == "" {
+	if content == "" && lang.String() != "" {
```

Comment:
This check didn't exist before I think? Fact is the asset change broke this, I'd rather understand why it broke than put a bandaid on the symptom.

Totally fair to descope that from this PR though.

Reply:
Filed story here: https://activestatef.atlassian.net/browse/DX-654. 

Do we want to keep this band-aid in the code for now or remove it and properly address it with the bug? From my short investigation into the issue this band-aid may end up being the solution. `packr` was likely also not finding the asset file, it just wasn't erroring either.

Reply:
A better check would be to see if lang is `Unknown`. The `Create()` function calls `language.MakeByName`, which can return `Unknown`.

Reply:
I tried that earlier and it did not work. However, I just realized that it didn't work because the language is not `Unknown` but `Unset`. Updated the check. Thanks!